apps/judicial-system/web/src/routes/CourtOfAppeal/Result/Result.tsx

```diff
@@ -18,15 +18,13 @@ import Conclusion from '@island.is/judicial-system-web/src/components/Conclusion
 import CaseFilesOverview from '../components/CaseFilesOverview/CaseFilesOverview'
 import CourtOfAppealCaseOverviewHeader from '../components/CaseOverviewHeader/CaseOverviewHeader'
 
-import { courtOfAppealResult as strings } from './Result.strings'
-import { courtOfAppealRuling as rulingStrings } from '../Ruling/Ruling.strings'
-
 import { useIntl } from 'react-intl'
-import { capitalize, formatDate } from '@island.is/judicial-system/formatters'
-import { CaseAppealRulingDecision } from '@island.is/judicial-system/types'
+import { capitalize } from '@island.is/judicial-system/formatters'
 import { titleForCase } from '../../Shared/SignedVerdictOverview/SignedVerdictOverview'
 import { core } from '@island.is/judicial-system-web/messages'
 import { appealCase } from '../AppealCase/AppealCase.strings'
```

Comment:
Please consider not importing Contentful strings from other components.

Generally, we want strings to accompany components and avoid sharing them. This way we are less likely to cause unintended changes when modifying strings in Contentful.

In this case, the strings belong to the `AppealCase` component, but are reused in three other components when constructing the `InfoCard` component. One possible change would be to simply pass `workingCase` to `InfoCard` and let `InfoCard` construct its info rather than passing it as `data`. Then the strings could be defined (duplicated) in `InfoCard`'s string file rather than importing them from `AppealCase`. This is only one possibility, but it would avoid importing strings from other components.